doc/ref_model/chapters/chapter03.md

```diff
@@ -185,4 +185,30 @@ Networking, alongside Compute and Storage, is an integral part of the Cloud Infr
 
+<p align="center"><img src="../figures/RM-Chapter4-Network-Model.png" alt="Network Model" Title="Network Model" width="65%"/></p>
+<p align="center"><b>Figure 3-5:</b> Network Model</p>
 
+Principles that should be followed during development and definition of Networking scopes for Reference Model, Reference Architectures and down to Reference Implementations and Reference Conformance test suites under CNTT:
+
+• Abstraction: A standardized layer between Virtualisation Layer domains and the Networking Hardware Layer resource pool hiding the implementation details of the Networking Hardware.
+
+    Note: Abstraction may be deployed in many different ways e.g. depending on target use case requirements, workload characteristics, different algorithm implementions of pipeline stages and available platforms. This includes Network Functions running on server nodes with or without programmable hardware acceleration, or networking support functions running on programmable network switches in the network.
+
+• Agnosticity : Define Network Fabric concepts and models that can carry any type of traffic in terms of:
+
+    Control, User and Management traffic types
+    Acceleration technologies supporting multiple types of Infrastructure deployments and Network Function workloads
+
+• Automation: Enable full automation, from Physical Fabric installation and provisioning to automation of VNF/CNF onboarding.
+
+• Openness: All Networking is based on open source or standardized APIs (NBI and SBI) and should enable integration of Open Source networking components (e.g. SDN controllers).
+
+• Programmability: Should enable a programmable forwarding plane controlled from a separately deployed control plane that abstracts into standardized functions and interfaces.
+
+• Scalability: Should be scalable to handle all traffic traverse North-South and East-West enabling small up to large deployments.
+
+• Workload diversity: Should be capable to p=to provide connectivity to any type of workloads, including VNF , CNF and BareMetal workloads.
+
+• Telco Grade: Should support deployments with Telco Grade characteristics requirements.
```

Comment:
Is it "Telco Grade characteristics" as in 'Telco grade workloads" or is it "Telso Grade requirements" (which requirements?)

Reply:
@pgoyal01, text updated